### PR TITLE
Update OpenAI blog credit usage changes

### PR DESCRIPTION
**Pull request by intelagense**
## Changes

- Add a note about the necessity to purchase credits for using GPT-3.5 API
- Remove outdated reference to free credits from OpenAI
- Update credit limit section to remove reference to free credits

### CODE REVIEW

File: projects/generate-a-blog-with-openai/generate-a-blog-with-openai.mdx [hunk 1]
```diff
@@ -63,7 +63,7 @@ Now we know what we'll be working with, let's build the program!
 
 ### OpenAI Account
 
-Before we do anything, we need an [OpenAI](https://openai.com/api) account. We'll need this account access to an API key that we can use to work with GPT-3.
+Before we do anything, we need an [OpenAI](https://openai.com/api) account. We'll need this account access to an API key that we can use to work with GPT-3. Note that OpenAI no longer offers free credits, so you'll also need to purchase credits to use the API.
```
Comment (Dusch4593) [inline projects/generate-a-blog-with-openai/generate-a-blog-with-openai.mdx:66]:
```suggestion
Before we do anything, we need an [OpenAI](https://openai.com/api) account. We'll need this to access an API key for using GPT-3. Note that OpenAI no longer offers free credits, so you'll also need to purchase credits to use the API.
```

Are we able to offer a little more in terms of linking to something that breaks down how pricing/credits works? Or maybe say what the minimum cost would be to go through this tutorial?

File: projects/generate-a-blog-with-openai/generate-a-blog-with-openai.mdx [hunk 2]
```diff
@@ -327,14 +325,14 @@ For GPT-3, the rate limit is 20 requests per minute. As long as we don't run the
...
-By this point, if you have been playing with the API nonstop, there's a chance that you might have exceeded the $18 limit. The following error is thrown when that happens:
+By this point, if you have been playing with the API nonstop, there's a chance that you might have exceeded your purchased credit limit. The following error is thrown when that happens:
 
 ```bash
 openai.error.RateLimitError:  
 You exceeded your current quota, please check your plan and billing details.
 ```
 
-If that's the case, go to OpenAI's [Billing overview](https://beta.openai.com/account/billing/overview) page and create a paid account.
+If that's the case, go to OpenAI's [Billing overview](https://beta.openai.com/account/billing/overview) page and purchase additional credits.
```
Comment (Dusch4593) [inline projects/generate-a-blog-with-openai/generate-a-blog-with-openai.mdx:335]:
```suggestion
If that's the case, go to OpenAI's [Billing overview](https://platform.openai.com/settings/organization/billing/overview) page and purchase additional credits.
```

More current link

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@intelagense left some feedback

cc: @asiqurrahman

File: projects/generate-a-blog-with-openai/generate-a-blog-with-openai.mdx (hunk 1 above)
Comment (intelagense) [reply projects/generate-a-blog-with-openai/generate-a-blog-with-openai.mdx:66]: The [pricing page](https://openai.com/api/pricing/) is full of tables and formulas that might be a bit much for learners that are just getting started. I'll add a quick note about the $5 minimum to keep it simple.

**Review by Dusch4593 — APPROVED** [review]
LGTM! 👍🏻 

I think we can merge this. cc: @asiqurrahman

**Pull request merged**